x-pack/plugin/esql/qa/testFixtures/src/main/resources/docs.csv-spec

```diff
@@ -21,6 +21,7 @@ avg_worked_seconds:long | birth_date:date | emp_no:integer | first_name:keyword
 docsEval
 // tag::eval[]
 FROM employees
+| SORT emp_no
```

Comment:
@abdonpijpelink this has a little impact on the docs, can you please check?
The problem with these tests is that SORT has to appear before KEEP (when it excludes the sort column) and LIMIT

Reply:
I think it's fine. It adds a few `SORT` commands to some examples in the docs, that didn't have them before, but that should not be a problem.